.github/workflows/examples-mbed.yaml

```diff
@@ -68,13 +68,19 @@ jobs:
               timeout-minutes: 10
               run: scripts/examples/mbed_example.sh -a=lighting-app -b=$APP_TARGET -p=$APP_PROFILE
 
+            - name: Build unit tests
+              timeout-minutes: 10
+              run: scripts/tests/mbed/mbed_unit_tests.sh -b=$APP_TARGET -p=$APP_PROFILE
+
             - name: Copy aside build products
               run: |
                   mkdir -p /tmp/output_binaries/$BUILD_TYPE-build
                   cp examples/lock-app/mbed/build-$APP_TARGET/$APP_PROFILE/chip-mbed-lock-app-example.hex \
                      /tmp/output_binaries/$BUILD_TYPE-build/lock-app-$APP_TARGET-$APP_PROFILE.hex
                   cp examples/lighting-app/mbed/build-$APP_TARGET/$APP_PROFILE/chip-mbed-lighting-app-example.hex \
                      /tmp/output_binaries/$BUILD_TYPE-build/lighting-app-$APP_TARGET-$APP_PROFILE.hex
+                  cp src/test_driver/mbed/build-$APP_TARGET/$APP_PROFILE/chip-mbed-unit-tests.hex \
```

Comment:
we do not need a copy of  unit tests for size comparisons

Reply:
@andy31415 We put unit tests binary to job artifact with integration testing in mind. It will be run on the target directly and the result will be checked by the serial port. Will this change be very problematic?

Reply:
artifacts will be auto-deleted by our bloat check script after we validate bloat, so actual storage will not be done for long.

I believe we should not add more artifacts at this point.

Reply:
Especially unit test artifacts: there is no value in bloat checking unit test artifacts.